released/packages/coq-mathcomp-algebra/coq-mathcomp-algebra.1.10.0/opam

```diff
@@ -0,0 +1,29 @@
+opam-version: "2.0"
+name: "coq-mathcomp-algebra"
+version: "1.10.0"
```

Comment:
I would remove the `name` and `version` from the package content as this is redundant with the file name.